fixes:
Building smbldap-config.8 from smbldap-config.cmd ...
pod2man: unable to format smbldap-config.cmd
--- smbldap-config.pl.orig      2014-10-04 08:33:06.524149179 +0000
+++ smbldap-config.pl   2014-10-04 08:33:07.748133877 +0000
@@ -557,3 +557,14 @@
 $mode=0600;
 chmod $mode,"$smbldap_bind_conf","$smbldap_bind_conf.old";
 
+=head1 NAME
+
+smbldap-config - Configure the smbldap-tools
+
+=head1 SYNOPSIS
+
+smbldap-config
+
+=head1 DESCRIPTION
+
+=cut
